web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -1306,6 +1358,38 @@ private Object validateItem(ImportItemDataBean itemDataBean, CrfBean crf, EventC
                 return new ErrorObj(NO_CHANGE, null);
 
             } else {
+            	if(isEventCrfCompleted(eventCrf)) {         		            		                
+                    
+					try {
+						QueryBean queryBean = new QueryBean();
+						queryBean.setType(QueryType.REASON.getName());
+						queryBean.setComment(this.DetailedNotes);
+						
+						QueryServiceHelperBean helperBean = new QueryServiceHelperBean();
+						helperBean.setItemData(itemData);
+						SubmissionContainer container = new SubmissionContainer(null, null, null, null, null, null, null, null);
+						container.setStudy(study);
```

Comment:
I would create new default constructor in SubmissionContainer class instead of passing null parameters.

Reply:
created